oneBarangay/static/assets/js/volt.js

```diff
@@ -15,166 +15,166 @@
 
 */
 
-"use strict";
-const d = document;
-d.addEventListener("DOMContentLoaded", function (event) {
+'use strict'
+const d = document
+d.addEventListener('DOMContentLoaded', function (event) {
   const swalWithBootstrapButtons = Swal.mixin({
     customClass: {
-      confirmButton: "btn btn-primary me-3",
-      cancelButton: "btn btn-gray",
+      confirmButton: 'btn btn-primary me-3',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/PrynsTag/oneBarangay/pullRequest?prid=7999497)
```suggestion
            confirmButton: "btn btn-primary me-3",
```